api/src/main/java/com/nicknathanjustin/streamercontracts/votes/VoteModelRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.nicknathanjustin.streamercontracts.votes;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface VoteModelRepository extends CrudRepository<VoteModel, UUID> {
+    VoteModel findByContractIdAndVoterId(UUID contractId, UUID voterId);
```

Comment:
Maybe rename this to userid? When I first read this I thought it was the "voteId"

Reply:
I have preference for this naming since it matches the database naming:

```voter_id UUID REFERENCES users (id),```